data/nsides/offsides/meta.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name: offsides
+description: OffSIDES is a database of individual drug side effect signals mined from the FDA's Adverse Event Reporting System. The innovation of OffSIDES
+    is that a propensity score matching (PSM) model is used to identify control drugs and produce better PRR estimates. In OffSIDES we focus on drug safety
+    signals that are not already established by being listed on the structured product label -- hence they are off-label drug side effects.
+targets:
+    - id: PRR
+      description: Proportional reporting ratio, PRR=(A/(A+B))/(C/(C+D))
+      type: continuous
+      names:
+          - Proportional reporting ratio
+    - id: PRR_error
+      description: Standard error of the PRR estimate
+      type: continuous
+      names:
+          - Proportional reporting ratio error
```

Comment:
do you think this is something the model should be able to predict? (I'm just curious)